### DISCUSSION TITLE
Why it shows errror/warning even project is created first time

### DISCUSSION THREAD

**Discussion by sarwan920** [nuxt/nuxt#20137] [Questions] [answered]
In my nuxt 3 project, whenever i use, useHead inside that over viewport property there comes weird error/warning, it happens when lang="ts" , if I remove that, it disappears.

what's the issue and how to fix it? I am attaching the screenshot below top.
![image](https://user-images.githubusercontent.com/47607107/230696967-acfb3faf-d540-478c-a08d-e97184d00449.png)

**Comment by Flowko** [comment, marked as answer]
its because `viewport` is not part of the outer accepted types for useHead https://nuxt.com/docs/api/composables/use-head#type
and u actually inserting that wrong, you should add it inside of `meta`, same thing for charset

```ts
useHead({
  ...
  meta: [
    {
      name: "viewport",
      content: "width=device-width, initial-scale=1",
    },
    {
      name: "charset",
      content: "utf-8",
    },
  ],
});
```

**Comment by sarwan920** [comment]
Thank you for the Help.
I was adding it as wrong configuration.
